projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -3,256 +3,102 @@
-- AI Assistant: hide image generation part [#30900]
-- AI Assistant: Import base Jetpack styles [#30881]
-- AI Assistant: introduce first approach of default prompt templates [#30941]
-- AI Assistant: Keep the option to accept title after asking for changes [#30942]
-- AI Assistant: rely on ai-assistant-feature to show upgrade banner [#31019]
-- AI Assistant: Remove correct spelling option in interactive mode [#30956]
-- AI Assistant: rename controls tooltips text [#30880]
-- AI Assistant: restore prompt rules removed by mistake [#30901]
-- AI Assistant: restore the multiple blocks edition feature [#30849]
-- AI Assistant: set fade effect when just requested completion [#30834]
-- AI Assistant: show the upgrade banner when the feature requires it [#30936]
-- AI Assistant: show upgrade banner when ai query endpoint responses with quota exceeded error [#30989]
-- AI Assistant: tweak assistant send button [#30896]
-- AI Assistant: tweak the upgrade banner [#31004]
-- AI Assistant: use AI icon in the block panels [#30853]
-- Change agency discount value on WPA [#30991]
-- Change CTA on backup recommendation card to inform about first year discount [#30895]
-- Connected block with backend and added remove_user_blogs option [#30639]
-- Enable Newsletters on self-hosred environments [#30952]
-- Handle panel changes when subscription module is inactive [#30958]
-- Jetpack AI: Remove check at AI Assistant block registration [#30960]
-- Jetpack Social: Add a notice to let users know Instagram is available [#30777]
-- Make the AI Assistant image generation button generate the image right awways instead of being a toggle [#30859]
-- Mastodon post preview [#30919]
-- Reading Settings: added a toggle for showing post views in the WordPress.com Reader [#30800]
+### Major Enhancements
+- Add an AI Assistant block, replacing the previous experimental AI Paragraph and AI Image blocks.
+- Donations, Paid Content and Payment Buttons blocks are now available with all Jetpack plans. A commission is charged for free plans.
+
+### Enhancements
+- Author Recommendations: Connected block with backend and added `remove_user_blogs` option. [#30639]
+- Blocks: Rename "Premium Content" block to "Paid Content" block. [#30775]
+- Contact Form: Improve spam filtering. [#30755]
+- Dashboard: Add Jetpack Boost module. [#30877]
+- Forms: Improve styling of response emails. [#30088]
+- Forms: Introduce Multiple Choice and Single Choice style variations. [#30319]
+- Forms: Update pattern modal default view to Grid. [#28906]
+- Image CDN: Replace Photon implementation with the image-cdn package. [#30050]
+- Maps: Add Mapkit maps. [#29098]
+- Modules list: Update to current styles. Visual refresh, if you will. [#30282]
+- My Plan: Display the correct plan for Jetpack Security and Backup 2-year plans. [#31013]
+- My Plan: Hide the "My Plan" on My Plan page if user has a paid project and a free plan. This way, users won't have much or conflicting information on that page. [#30804]
```

Comment:
```suggestion
- My Plan: Hide the "My Plan" on My Plan page if user has a paid product and a free plan. [#30804]
```

Reply:
Huh, I thought I cleaned that one up already.